src/main/scala/zio/zmx/diagnostics/ZMXServer.scala

```diff
@@ -43,22 +44,26 @@ object Codec {
 object RequestHandler {
 
   def handleRequest(
```

Comment:
This is only called in `whenIsReadable` so we could move it in into the `make` method. Then we wouldn't even need to pass it as a parameter since the `getFiberDumps` effect would already be in scope there.